include/GafferImage/ImagePlug.h

```diff
@@ -135,6 +136,26 @@ class ImagePlug : public Gaffer::ValuePlug
 		IECore::MurmurHash imageHash() const;
 		//@}
 
+		/// Utility class to scope a temporary copy of a context,
+		/// with tile/channel specific variables removed. This can be used
+		/// when evaluating plugs which must be global to the whole image,
+		/// and can improve performance by reducing pressure on the hash cache
+		struct GlobalScope : public Gaffer::Context::EditableScope
+		{
+			GlobalScope( const Gaffer::Context *context );
+		};
+
+		/// Utility class to scope a temporary copy of a context,
```

Comment:
This comment seems to apply to the GlobalScope rather than the ChannelDataScope.